views/login.ejs

```diff
@@ -1,11 +1,28 @@
 <%- include("./partials/header") %>
 
-<h1>Login</h1>
+<div class="container">
+    <div class="row">
+        <div class="col">
+            <h1 style="text-align: center; margin-top: 30px;">Login</h1>
+            <div style="width: 30%; margin: 30px auto;">
+                <form action="/login" method="POST">
+                    <div class="form-group">
+                        <label for="exampleInputUsername1">Username</label>
+                        <input class="form-control" type="text" name="username" id="exampleInputUsername1">
```

Comment:
Why the ID is `exampleInputUsername1`? How it's an example?
You might not need the number at the end of the ID too.
Also - I'd suggest using kebab-case IDs.

Reply:
It would be better to write login-input-username?

Reply:
It could be just `username` or `username-input`.